unit-tests/live/options/test-presets.py

```diff
@@ -0,0 +1,51 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:donotrun:gha
```

Comment:
What does it means?
Where would this test run?

Aren't we missing 
#device d400*
#device D500*? (not sure it will work)

Reply:
Beside that question, great test!!

Reply:
I checked,
LibCI: -D- test-live-options-presets is not live; skipping
GHA - -D- test-live-options-presets is marked do-not-run; skipping

GHA is not running live tests IMO.
You should add the matching #device keywords